packages/react-docsite-components/src/components/PageHeader/PageHeader.tsx

```diff
@@ -39,15 +58,47 @@ const getStyles: IStyleFunction<IPageHeaderStyleProps, IPageHeaderStyles> = prop
 const getClassNames = classNamesFunction<IPageHeaderStyleProps, IPageHeaderStyles>();
 
 const PageHeaderBase: React.FunctionComponent<IPageHeaderProps> = props => {
-  const { pageTitle = 'Page title', pageSubTitle, theme } = props;
-  const styles = getClassNames(getStyles, { theme });
+  const { className, pageTitle = 'Page title', pageSubTitle, theme, versionSwitcherDefinition } = props;
+  const styles = getClassNames(getStyles, { className, theme });
+
+  const versionOptions: IContextualMenuItem[] | undefined =
+    versionSwitcherDefinition && versionSwitcherDefinition.versions
+      ? versionSwitcherDefinition.versions.map(version => ({
+          key: version,
+          text: version,
+          checked: version === versionSwitcherDefinition.currentVersion,
+        }))
+      : undefined;
 
   return (
-    <header className={css(styles.root, props.className)}>
+    <header className={styles.root}>
       <h1 className={styles.title}>
         {pageTitle}
         {pageSubTitle && <span className={styles.subTitle}>{pageSubTitle}</span>}
       </h1>
+      {versionSwitcherDefinition && versionOptions && (
```

Comment:
We should probably hide this on mobile, at least phones. Not exactly sure how to handle the breakpoints here, since within the UHF "mobile" range (up to 768px) the dropdown is bad on the smaller end and fine on the larger end (then bad again up to around 900px).

Medium-sized phone (375px):
![image](https://user-images.githubusercontent.com/5864305/99318744-2aabd900-281d-11eb-8f4b-762833c7d81a.png)

Tablet vertical (767px--this is just below the UHF "mobile" breakpoint of 768px, since exactly 768px seems to have some bugs, unrelated to your PR)
![image](https://user-images.githubusercontent.com/5864305/99318821-50d17900-281d-11eb-9458-12a2be3ebf0e.png)

800px
![image](https://user-images.githubusercontent.com/5864305/99318626-e7516a80-281c-11eb-8505-212e1638edbf.png)

800px with abbreviated version
![image](https://user-images.githubusercontent.com/5864305/99319011-a73eb780-281d-11eb-8ff1-d319d9a0b3ba.png)